sample.env

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Make sure you've got a .gitignore file that you're adding this type of file to so it doesn't get pushed up.

Reply:
I do have gitignore. In this case I do want sample.env to be in the repo and expect another user/tester to copy it to a local .env file. Good eye and thanks for looking this over!